Matrix/MatrixTest.java

```diff
@@ -0,0 +1,95 @@
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import org.junit.Test;
+
+public class MatrixTest {
+
+	/*
+	 * happy test cases
+	 */
+	@Test
+	public void addElementsHappyTestCase() {
+		Matrix object = new Matrix(2, 2);
```

Comment:
Matrix creation and adding elements is common in several tests, this can be moved in @before or can be done in a separate method